echopype/convert/set_groups_ek60.py

```diff
@@ -59,8 +59,22 @@ def __init__(self, *args, **kwargs):
         super().__init__(*args, **kwargs)
 
         self.old_ping_time = None
+
+        # obtain sorted channel dict in ascending order
+        channels = list(self.parser_obj.config_datagram["transceivers"].keys())
+        channel_ids = {
+            ch: self.parser_obj.config_datagram["transceivers"][ch]["channel_id"] for ch in channels
+        }
+        self.sorted_channel = dict(sorted(channel_ids.items(), key=lambda item: item[1]))
+
+        # obtain corresponding frequency dict from sorted channels
+        self.freq = [
+            self.parser_obj.config_datagram["transceivers"][ch]["frequency"]
+            for ch in self.sorted_channel.keys()
+        ]
```

Comment:
I think it's great that these are in init!

Reply:
Me too! I was going to provide something like this for EK80, but I decided to move on. However, I did see that you think it would be worth it to provide a similar thing for EK80. So, I will go ahead and implement it after I have gone through AZFP.